app/src/main/java/com/benavalli/arcmovies/api/config/BuildConstants.kt

```diff
@@ -0,0 +1,12 @@
+package com.benavalli.arcmovies.api.config
+
+class BuildConstants {
+    companion object {
+        val BASE_IMAGE_URL = "https://image.tmdb.org/t/p/w200"
+        val BASE_IMAGE_POSTER_URL = "https://image.tmdb.org/t/p/w500"
+        val BASE_API_URL = "https://api.themoviedb.org/3/"
+        val API_KEY = "1f54bd990f1cdfb230adb312546d765d"
```

Comment:
I really enjoy the way you use the interceptor to call the API constants, for me its the best pratictice.

But a safer way is calling by the cradle, like that:

https://technobells.com/best-way-to-store-your-api-keys-for-your-android-studio-project-e4b5e8bb7d23

Reply:
good point, i put the key in the gradle